self-hosted/migration/schema-then-data.md

```diff
@@ -349,7 +320,7 @@ don't include deleted raw data.
     (1 row)
     ```
 
-1.  Connect to your Timescale Cloud database:
+1.  Connect to your Timescale database:
 
     ```bash
     psql "postgres://tsdbadmin:<CLOUD_PASSWORD>@<CLOUD_HOST>:<CLOUD_PORT>/tsdb?sslmode=require"
```

Comment:
This script also references cloud (but is in self-hosted)

Reply:
```suggestion
    psql "postgres://tsdbadmin:<PASSWORD>@<HOST>:<PORT>/tsdb?sslmode=require"
```